baseplate/experiments/providers/base.py

```diff
@@ -0,0 +1,24 @@
+
+class ExperimentInterface(object):
```

Comment:
Oh I'm totally on board with having an abstract base class, just suggesting the Interface suffix be dropped.